src/Symfony/Component/Notifier/Bridge/Telegram/README.md

```diff
@@ -14,6 +14,27 @@ where:
  - `TOKEN` is your Telegram token
  - `CHAT_ID` is your Telegram chat id
 
+Interacting with local API server instead of official Telegram API
+------------------------------------------------------------------
+
+If such a case is needed, you can replace the `default` keyword in the DSN
+with the desired domain/IP address of your local API server. You may also want to
+disable the bridge's default behavior of using `https` protocol as local API servers
+can only accept `http` traffic.
+
+Example:
+```
+TELEGRAM_DSN=telegram://TOKEN@localhost:5001?channel=CHAT_ID&disable_https=1
```

Comment:
In AmazonSns, this option is named `sslmode`. I don't know if this name is better, but I think it's better to be consistent between adapters. https://github.com/symfony/symfony/blob/9ad4353a8cf349c3b431c163af0b1dd9a29c33a2/src/Symfony/Component/Notifier/Bridge/AmazonSns/AmazonSnsTransportFactory.php#L34

This would become:
```suggestion
TELEGRAM_DSN=telegram://TOKEN@localhost:5001?channel=CHAT_ID&sslmode=disable
```

Reply:
I know you've already changed the name in response to our request, but perhaps the consistency with other bridges hadn't been examined. https://github.com/symfony/symfony/pull/57769#discussion_r1683918796